notifications_utils/sanitise_text.py

```diff
@@ -17,6 +17,8 @@ class SanitiseText:
         '\t': ' ',  # TAB
     }
 
+    WELSH_NON_GSM_CHARACTERS = 'ÂâÊêÎîÔôÛûŴŵŶŷ'
```

Comment:
i think this should be a set (same as `ALLOWED_CHARACTERS`)

Reply:
also, I think it should be in the `SanitiseGSMandWelsh` class rather than the base class

Reply:
👍I've changed this to be a set in the `SanitiseSMS` class